lib/ui/pages/profile.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:pispapp/controllers/app/auth_controller.dart';
+import 'package:pispapp/controllers/ephemeral/profile_controller.dart';
+import 'package:pispapp/ui/widgets/bottom_button.dart';
+import 'package:pispapp/ui/widgets/title_text.dart';
+import 'package:get/get.dart';
+
+class Profile extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      resizeToAvoidBottomPadding: false,
+      body: Container(
+        padding: const EdgeInsets.symmetric(
+          horizontal: 20,
+        ),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            const Padding(
+              padding: EdgeInsets.fromLTRB(
```

Comment:
can you make this a single line by removing the trailing comma? same for other similar parts of the code
you can try doing `CTRL+F` here to look at all other parts

Reply:
addressed in https://github.com/mojaloop/pisp-demo-app-flutter/pull/12